libs/image-utils/src/jest_pdf_snapshot.ts

```diff
@@ -33,7 +33,7 @@ export async function toMatchPdfSnapshot(
     typeof received === 'string' ? await readFile(received) : received;
   const pdfPages = pdfToImages(pdfContents, { scale: 200 / 72 });
   for await (const { page, pageNumber } of pdfPages) {
-    const imageBuffer = toImageBuffer(page);
+    const imageBuffer = await encodeImageData(page, 'image/png');
```

Comment:
Somehow this is causing this sort of failure:

![image](https://github.com/user-attachments/assets/27b7fe86-4808-478b-bfff-25e5b44ae3bb)

Feels like a race condition of some sort.

Reply:
I reverted this change to avoid this problem.